### PR TITLE
Change send_icmp to use OS default ping size

### PR DESCRIPTION
**Pull request by MosheSheena**
Changed send_icmp function to use default host OS ping size
in case a size is not passed to the fuction

### CODE REVIEW

File: rrmngmnt/network.py [hunk 1]
```diff
@@ -559,8 +559,8 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
             bool: True/false
         """
         cmd = ["ping", dst, "-c", count, "-s", size]
```
Comment (myakove) [inline rrmngmnt/network.py]: You need to remove the size from here.

File: rrmngmnt/network.py [hunk 2]
```diff
@@ -559,8 +559,8 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
...
-        if size != "1500":
-            cmd.extend(["-M", "do"])
+        if size is not None:
+            cmd.extend(["-s", str(size), "-M", "do"])
```
Comment (myakove) [inline rrmngmnt/network.py]: From the docstring size is str so no need to call str(size)

File: rrmngmnt/network.py [hunk 3]
```diff
@@ -559,8 +559,8 @@ def send_icmp(self, dst, count="5", size="1500", extra_args=None):
             bool: True/false
         """
         cmd = ["ping", dst, "-c", count, "-s", size]
-        if size != "1500":
-            cmd.extend(["-M", "do"])
+        if size is not None:
```
Comment (myakove) [inline rrmngmnt/network.py]: size is a value so you can remove "is not None".

File: rrmngmnt/network.py (hunk 1 above)
Comment (MosheSheena) [reply rrmngmnt/network.py]: Right! I forgot it (0_0)... will be removed.

**Comment by lukas-bednar** [comment]
looks good, I am afraid that you need to add test to keep coverage of this project.

**Pull request merged**